tests/ckeditor/ckeditor.js

```diff
@@ -48,6 +48,14 @@ describe( 'create', function() {
 		} );
 	} );
 
+	it( 'should set configurations on the new editor', function() {
+		var CKEDITOR = modules.ckeditor;
+
+		return CKEDITOR.create( content, { test: 1 } ).then( function( editor ) {
+			expect( editor.config.test ).to.equals( 1 );
```

Comment:
I'm almost certain we agreed to use the correct form (`to.equal`) right after this issue appeared for the first time.